src/lib/eZ/RichText/Resources/stylesheets/xhtml5/edit/docbook.xsl

```diff
@@ -787,6 +787,24 @@
     <xsl:value-of select="translate( substring-before( substring-after( concat( substring-after( $style, $property ), ';' ), ':' ), ';' ), ' ', '' )"/>
   </xsl:template>
 
+  <xsl:template name="extractTextAlignValue">
+    <xsl:param name="style"/>
+    <xsl:param name="property"/>
+    <xsl:variable name="alignValue">
+      <xsl:call-template name="extractStyleValue">
+        <xsl:with-param name="style" select="$style"/>
+        <xsl:with-param name="property" select="$property"/>
+      </xsl:call-template>
+    </xsl:variable>
+
+    <xsl:choose>
+      <xsl:when test="$alignValue = 'left' or $alignValue = 'center' or $alignValue = 'right' or $alignValue = 'justify'">
+        <xsl:value-of select="$alignValue"/>
+      </xsl:when>
+      <xsl:otherwise>left</xsl:otherwise>
```

Comment:
So, I here a default value of `left` is used when the value provided is not supported.

This means that we silently changes whatever user provided. However, the validation errors provided by libxml is anyway not very informative so I think this is okay